### PR TITLE
Tweak readme

### LINKED ISSUE #16: rewrite README
- [x] Badges
- [x] Remove `Frameworks` section
- [x] Add short description
- [x] Replace images with local ones
- [x] Remove `Credits` section
- [x] Add copyright notice (without year) and Zuul credits to `License` section
- [x] And most importantly: #10

### LINKED ISSUE #26: Give credit to Sauce Labs
A request from Sauce Labs:

> We would also like to request that you give credit to Sauce Labs, via either a text link and/or logo and link on your repo and project website, should there be a pubic-facing site; I've attached logo artwork and a MD file with text. If you'd like to write a blog post about your project we're happy to publish it on the Sauce Labs website, and shoutouts on Twitter are always appreciated! 

Files (SVG, PNG and MD files): [sauce labs assets.zip](https://github.com/airtap/airtap/files/1732165/sauce.labs.assets.zip)

### LINKED ISSUE #27: Update screenshot
A request from Sauce Labs:

> If possible, it would be great if you could also update this screenshot to reflect our current branding:
> image

### PR DESCRIPTION
**Pull request by vweevers**
Closes #16, closes #26. Squash on merge.

### CODE REVIEW

**Comment by vweevers** [comment]
Hm: https://github.com/airtap/airtap/issues/27#issuecomment-366503722

File: README.md [hunk 1]
```diff
@@ -2,7 +2,9 @@
 
 > Test your JavaScript in 800+ browsers.
 
+[![npm status](http://img.shields.io/npm/v/airtap.svg)](https://www.npmjs.org/package/airtap) [![node](https://img.shields.io/node/v/airtap.svg)](https://www.npmjs.org/package/airtap)
```
Comment (ralphtheninja) [inline README.md]: Nitpick: Mind putting the node badge on the next line?

**Comment by vweevers** [comment]
Don't merge yet, I want to find something better for the red browser matrix.

**Comment by vweevers** [comment]
Nevermind, I'll merge, but I'm keeping the screenshot issue open. See https://github.com/airtap/airtap/issues/27#issuecomment-366506917.

**Comment by vweevers** [comment]
Note: the [build failure](https://travis-ci.org/airtap/airtap/jobs/342970420) is unrelated:

```
The command "eval git fetch origin +refs/pull/34/merge: " failed. Retrying, 2 of 3.
```

**Pull request merged**